src/lib/Preview.js

```diff
@@ -121,6 +125,9 @@ class Preview extends EventEmitter {
     /** @property {PreviewUI} - Preview's UI instance */
     ui;
 
+    /** @property {Logger} - Preview's LogLayer instance */
+    logger;
```

Comment:
This naming is a little confusing. We call it `logger` and it's an instance of `Logger` but it's actually an instance of `LogLayer`?

Reply:
I completely missed this. Updated